packages/xod-func-tools/src/index.js

```diff
@@ -332,6 +332,28 @@ export const invertMap = def(
   )
 );
 
+/**
+ * Returns the list of list of strings.
+ * Removes all duplicates from the subsequent list
+ * on the basis of already filtered values lists.
+ *
+ * E.G.
+ * [['a', 'b', 'c'], ['b','c','d'], ['a','d','e']]
+ * will become
+ * [['a', 'b', 'c'], ['d'], ['e']]
+ */
+export const uniqLists = def(
+  'uniqLists :: [[String]] -> [[String]]',
+  R.reduce(
+    (acc, nextList) => R.compose(
+      R.append(R.__, acc),
+      R.reject(R.__, nextList),
+      isAmong,
+      R.flatten
+    )(acc),
+    []
+  )
+);
```

Comment:
`flatten` could make our list a bit too flat😁, `unnest` is probably better.
Also, why not use `without`? Something like 
```
R.reduce(
  (acc, nextList) => R.append(R.without(R.unnest(acc), nextList), acc),
  []
)
```